tests/test_scheduler.py

```diff
@@ -714,8 +714,8 @@ def test_full_loop_no_noise(self):
         result_sum = torch.sum(torch.abs(sample))
         result_mean = torch.mean(torch.abs(sample))
 
-        assert abs(result_sum.item() - 14379591680.0) < 1e-2
-        assert abs(result_mean.item() - 18723426.0) < 1e-3
+        assert abs(result_sum.item() - 14379591680.0) < 2.5e3
```

Comment:
Can we add a comment here that says that the results are flaky depending on GPU and this is why we have such high values?

Reply:
Actually upon taking a second look I think the reason is that we don't make the test deterministic with generators - both DDPM and Score-VE schedulers make use of `torch.randn(...)`